### PR TITLE
Iss1 create poem page

### PR DESCRIPTION
**Pull request by tynanook**
@yuens1002 @JBigs206 
This is the branch for iss1, creating a poem page. [iss1-create-poem-page](https://github.com/suwebdev/proud-dew/issues/1).
I do not know what the .idea/....xml file is.
The page is consistent with our website design. I arbitrarily populated the page with a Robert Frost poem. The reviews and the review button at the bottom section of the page is related to issue 2 (and to a lesser degree, issue 3).

### CODE REVIEW

**Comment by tynanook** [comment]
I added @yuens1002 and @JBigs206 as assignees, for review as they are familiar with the navigation of the website, which this page will is dependent upon.

**Comment by tynanook** [comment]
This pull request is waiting on comments for at least 2 members of the team before it can be merged. @yuens1002 @JBigs206 @wibblywink

File: poem.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+            Yet knowing how way leads on to way,<br>
+            I doubted if I should ever come back.<br>
+            <br>
+            I shall be telling this with a sigh<br>
+            Somewhere ages and ages hence:<br>
+            Two roads diverged in a wood, and I—<br>
+            I took the one less traveled by,<br>
+            And that has made all the difference.<br>
+        </p>
+
```
Comment (ghost) [inline poem.html:124]: Add a comment here so it's clear that we change from the poem section to the review. I suggest a 'Reader reviews' comment in a new line.

**Comment by yuens1002** [comment]
the page is broken - no style. pls first merge from one of my branches when you're in your branch.  do, git checkout "your branch name", then git merge "one of my remote branches".  you'll pull all the current site markup (pages) and css which you can use to apply styles to your poem.html file.  you can model your poem.html page off of the index.html to get all the styles to show up.  try to avoid modification of files you didn't create, then commit/push all your changes back to remote.  when merging, all your commits can be collapsed into one single merge.

is the xml file really necessary for the site, is there a reason why it's included in this issue?

**Comment by tynanook** [comment]
@yuens1002 @wibblywink @JBigs206 
I merged the master pages into this branch to allow easier viewing of the 'poem.html' page. 
I modified the 'poems.html' to allow functional viewing of the 'poem.html' page. To view the 'poem.html' page:  from the homepage, click on the 'poems.html' --> 'titles' in the navbar menu, then either click on 'Aaron' or 'Abacus'

There review section has a title and comments added in the file.
I removed unnecessary comments. 
I'm not sure where the .xml file originated, but it is now deleted.

**Comment by JBigs206** [comment]
@tynanook Looks good. I will link the poems in my dropdown menu to your poem.html page.

File: poem.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      <!-- Example row of columns -->
+      <div class="row">
+        <div class="col-md-4">
+			  <hr>
+            <img src="./img/360px-Jb_modern_frost_2_e%20(1).jpg" class="img-responsive" alt="Author Image">
+
+            <hr>
+              <p class="poem-pg-author-bio">Robert Frost was born in San Francisco, California, to journalist William Prescott Frost, Jr., and Isabelle Moodie. His mother was a Scottish immigrant, and his father descended from Nicholas Frost of Tiverton, Devon, England, who had sailed to New Hampshire in 1634 on the Wolfrana.</p>
+        </div>
+        <div class="col-md-8 "poem-pg-poem-text">
```
Comment (yuens1002) [inline poem.html:96]: take out the extra ", should be

`<div class="col-md-8 poem-pg-poem-text">`

File: poems.html [hunk 3]
```diff
@@ -109,10 +109,10 @@
         <div class="col-md-8">
           <h2 class="visible-md-block visible-lg-block">Title<br><br></h2>
 				
-          <h2>A - C</h2><p><a href="#">Aaron</a>
+          <h2>A - C</h2><p><a href="./poem.html">Aaron</a>
```
Comment (yuens1002) [inline poems.html:112]: `<a href="poem.html">Aaron</a>`

this is how poem.html should be linked, b/c all of our html files sit at the same root level.

File: poems.html [hunk 4]
```diff
@@ -109,10 +109,10 @@
...
 <br>BY GEORGE HERBERT
 <br>Holiness on the head,</p>
-<p><a href="#">Abacus</a>
+<p><a href="./poem.html">Abacus</a>
```
Comment (yuens1002) [inline poems.html:115]: the same goes here

**Comment by yuens1002** [comment]
i would leave out workspace.xml on your next commit the file don't seemed to be linked or used.  would be nice to include the side nav in your layout for consistency, and to allow users to navigate back or to another section within poems.  i left inline comments regarding to your poem.html.  great work! pls merge asap.

**Pull request merged**